docs/cmake-settings.md

```diff
@@ -19,6 +19,8 @@ Options that support substitution, in the table below, allow variable references
 | `cmake.cmakeCommunicationMode` | Specifies the protocol for communicating between the extension and CMake | `automatic` | no |
 | `cmake.configureArgs` | Arguments to CMake that will be passed during the configure process. Prefer to use `cmake.configureSettings` or [CMake variants](variants.md).</br> Do not pass `-D` arguments using this setting. | `[]` (empty array-no arguments) | yes |
 | `cmake.configureEnvironment` | An object containing `key:value` pairs of environment variables, which will be passed to CMake only when configuring.| `null` (no environment variable pairs) | yes |
+| `cmake.configureOnEdit` | Automatically configure CMake project directories when cmake.sourceDirectory or CMakeLists.txt content are saved. | `true` | no |
```

Comment:
Should cmake.sourceDirectory be `cmake.sourceDirectory` (to be consistent with other usages in the file)?